integrations/hubspot/test/index.test.js

```diff
@@ -37,13 +38,15 @@ describe('HubSpot', function() {
         .global('_hsq')
         .global('hbspt')
         .option('loadFormsSdk', false)
+        .option('loadChatWidget', false)
         .option('portalId', null)
     );
   });
 
   describe('before loading', function() {
     beforeEach(function() {
       analytics.stub(hubspot, 'load');
+      analytics.stub(window.HubSpotConversations.widget, 'load');
```

Comment:
why do you want to change the behaviour for each test case?Why not limit it to the current test case that you are writing?

Reply:
I found that every stub is created in beforeEach(). that's why I did this.

Reply:
try to see if you can have a before step in our test case.That way the stub will be limited to only our test run